Packs/SecneurXAnalysis/Playbooks/Detonate_File_-_SecneurX_Analysis_README.md

```diff
@@ -53,8 +59,10 @@ This playbook does not use any scripts.
 | SecneurXAnalysis.Report.FileDeleted | List of File deletions observed in the analyzed sample | string |
 | SecneurXAnalysis.Report.FileModified | List of File changes observed in the analyzed sample | string |
 | SecneurXAnalysis.Report.IOC | List of IOC's observed in the analyzed sample | string |
-| SecneurXAnalysis.Report.Status | Analysis queued sample state | string |
+| SecneurXAnalysis.Report.Status | Analysis queued sample state | String |
```

Comment:
```suggestion
| SecneurXAnalysis.Report.Status | Analysis queued sample state. | String |
```
Why is this "String" type uppercase, when all the other "string" types are lower case?